std/container/slist.d

```diff
@@ -123,6 +123,19 @@ struct SList(T)
         return n;
     }
 
+    private static Node * findNodeByValue(Node * n, T value)
```

Comment:
Nit: style: `Node*`

Reply:
Was trying to maintain the general style of the Slist implementation. I will update it.

Reply:
As mentioned in https://wiki.dlang.org/Contributing_to_Phobos:

> Be critical when looking at Phobos code (some parts of Phobos are a bit older and have some relics...)